capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java

```diff
@@ -35,7 +35,7 @@
 /** Servlet that returns Perspective scoring. */
 @WebServlet("/call_perspective")
 public class CallPerspectiveServlet extends HttpServlet {
-  private static final String URL = "https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=AIzaSyCNknbH7wekG_bz1RcP1muXy9plNQhZaAY";
```

Comment:
Unfortunately this commit diff means that we'll likely want to generate a new API key to make sure this isn't possibly abused.

Reply:
Agreed!